Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -1291,19 +1369,49 @@ MediaPlayer::MediaKeyException MediaPlayerPrivateGStreamerBase::addKey(const Str
     return MediaPlayer::NoError;
 }
 
+void MediaPlayerPrivateGStreamerBase::trimInitData(String keySystemUuid, const unsigned char*& initDataPtr, unsigned &initDataLength)
+{
+    if (!m_initDatas.contains(keySystemUuid)) {
+        GST_TRACE("we don't have an initData for %s", keySystemUuid.utf8().data());
+        return;
+    }
+
+    Vector<uint8_t> storedInitData = m_initDatas.get(keySystemUuid);
```

Comment:
I can try it but I need so ensure I can use this to search on the contained data easily.